tests/parametric/test_otel_span_methods.py

```diff
@@ -484,29 +484,63 @@ def test_otel_span_reserved_attributes_overrides(self, test_agent, test_library)
         assert "analytics.event" not in span["meta"]
 
     @missing_feature(context.library < "java@1.25.0", reason="Implemented in 1.25.0")
-    @missing_feature(context.library == "nodejs", reason="Not implemented")
     @missing_feature(context.library == "dotnet", reason=".NET dosn't treat 'something-else' as a valid input here.")
+    @missing_feature(context.library == "nodejs", reason="Not implemented")
     @missing_feature(context.library == "python", reason="Not implemented")
     @missing_feature(context.library == "python_http", reason="Not implemented")
     @pytest.mark.parametrize(
         "analytics_event_value,expected_metric_value",
-        [
-            ("true", 1),
-            ("TRUE", 1),
-            ("True", 1),
-            ("false", 0),
-            ("False", 0),
-            ("FALSE", 0),
-            ("something-else", 0),
-            (True, 1),
-            (False, 0),
-        ],
+        [("true", 1), ("TRUE", 1), ("True", 1), ("false", 0), ("False", 0), ("FALSE", 0), (True, 1), (False, 0),],
+    )
+    def test_otel_span_basic_reserved_attributes_overrides_analytics_event(
+        self, analytics_event_value: Union[bool, str], expected_metric_value: Union[int, None], test_agent, test_library
+    ):
+        """
+            Tests the analytics.event reserved attribute override with basic inputs
+        """
+        run_otel_span_reserved_attributes_overrides_analytics_event(
+            analytics_event_value=analytics_event_value,
+            expected_metric_value=expected_metric_value,
+            test_library=test_library,
+            test_agent=test_agent,
+        )
+
+    @irrelevant(context.library == "java", reason="The expected metric value is zero")
```

Comment:
For this one, the reason does not explain very well why it's irrelevant : 

* does it mean the test is not good because *The expected metric value is zero* -> so we need to update the test
* does it mean that those tracer decided to set the metric to zero -> the reason must make claer it's an educated choice
* the expected metric is really zero, but those tracer are failing ? -> it must be decorated with `@bug`
* other interpretation I may not see ?

Reply:
With respect to the RFC, it would be the second one. Here is what comes to my mind:
- `Choose to set the metric value to zero for parseBool-rejected values` 
- `Chose zero metric for parseBool-rejected inputs`
- `Set metric to zero when rejected by parseBool`

Reply:
WDYT  about a message making  crystal clear the fact it's a design choice ? Something like  `Java tracer decided to ...`

Reply:
Alright, I changed the message to `<language> tracer decided to always set _dd1.sr.eausr: 1 for truthy analytics.event inputs, else 0", which imo, clearly says what's going on, although it could be argued that "truthy" is vague (and it is).